src/mat/_mat/mat_ops.rs

```diff
@@ -65,7 +65,7 @@ use std::result::Result;
 /// ```
 impl<T> Add for Matrix<T>
```

Comment:
It seems like the test for adding `Matrix<T>` does not work. I'll see if I can find the error.

Reply:
Found the issue, the Matrix needed a type.